<launch>

  <arg name="simulation" default="false" />
  <arg name="use_learning" default="false" />
  <arg name="use_motion_cost" default="false" />
  <arg name="domain" default="$(find bwi_kr_execution)/domain/" />
  <arg name="executor" default="plan_executor_node" />

  <param name="bwi_kr_execution/domain_directory" type="str" value="$(arg domain)" />

  <node name="plan_executor" pkg="bwi_kr_execution" type="$(arg executor)" output="screen">
    <!-- launch-prefix="gdb -ex run -args" -->
    <param name="simulation" type="bool" value="$(arg simulation)" />
    <param name="use_learning" type="bool" value="$(arg use_learning)" />
    <param name="use_motion_cost" type="bool" value="$(arg use_motion_cost)" />

  </node>

</launch>
